src/main/java/io/supertokens/storage/mysql/queries/PasswordlessQueries.java

```diff
@@ -181,7 +181,8 @@ public static void createCode(Start start, PasswordlessCode code)
             } catch (SQLException e) {
 
                 String message = e.getMessage();
-                if (message.contains("foreign key constraint fails") && message.contains("passwordless_codes")
+                if (message.contains("foreign key constraint fails")
```

Comment:
If you see other places, we do `e.getMessage().contains("foreign key") && ...`. So why not do that here too? Just to be consistent.

Reply:
I got this from : "String message = e.actualException.getMessage();" and I felt this is a much cleaner way to do it. Let me know what needs to be done.

Reply:
Proceeding with updating just the ones with message = e.getMessage(); Will leave the e.actualException.getMessage() to be updated in another PR.

Reply:
I should have been clearer.. My issue is with using "foreign key constraint fails" as the match criteria instead of just "foreign key"

Reply:
oh alright. done.